Legacy/bonej/pom.xml

```diff
@@ -1,170 +1,160 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-	<parent>
-		<groupId>org.bonej</groupId>
-		<artifactId>pom-bonej</artifactId>
-		<version>7.0.2-SNAPSHOT</version>
-		<relativePath>../..</relativePath>
-	</parent>
+    <parent>
+        <groupId>org.bonej</groupId>
+        <artifactId>pom-bonej</artifactId>
+        <version>7.0.1-SNAPSHOT</version>
```

Comment:
Are you sure about changing this? Pretty sure we are currently on 7.0.2-SNAPSHOT